### PR TITLE
Updated documentation with networking diagrams for different cluster flavors

### PR DESCRIPTION
**Pull request by lintongj**
**What this PR does / why we need it**:

Updated documentation with networking diagrams for different cluster flavors

Signed-off-by: Lintong Jiang <lintongj@vmware.com>

**Which issue(s) this PR fixes**:

N/A

**Special notes for your reviewer**:
N/A

**Does this PR introduce a user-facing change?**:

```release-note
N/A
```

### CODE REVIEW

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/494/

**Comment by lintongj** [comment]
Hey @cormachogan, would you please help review it? Thanks

File: docs/vanilla.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ of vSphere CSI Driver.
 * Meet the prerequisites of Virtual Disk Development Kit (VDDK).
   * Please refer to [Credentials and Privileges for VMDK Access](https://code.vmware.com/docs/11750/virtual-disk-development-kit-programming-guide/GUID-8301C6CF-37C2-42CC-B4C5-BB1DD28F79C9.html)
 of VDDK Programming Guide. **Note: please apply privileges at the vCenter Server level**.
-  * Please make sure port **902** is opened on each ESXi host based on the requirement of the VDDK NBD transport mode.
+  * Please make sure open port **902** on any ESXi that hosts Kubernetes node VMs. This port requirement is specific to the NBD transport mode of VDDK. Below is an example of networking setup in Vanilla cluster on vSphere.
```
Comment (xing-yang) [inline docs/vanilla.md:29]: nit: "make sure open" -> "make sure to open"

**Comment by svcbot-qecnsdp** [comment]
Congratulations !! 

All the pre-check tests passed
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/508/

**Review by xing-yang — APPROVED** [review]
LGTM

**Pull request merged**